R/standard_panels.R

```diff
@@ -26,6 +26,21 @@ support_panel <- function(
     publication_stub = "",
     alt_href = NULL,
     form_url = NULL) {
+  # Check that the team_email is a valid email
+
+  is_valid_email <- function(email) {
+    grepl(
+      "\\<[A-Z0-9._%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,}\\>",
```

Comment:
Can we be more stringent than this? Should we mandate that it should specifically be an @education.gov.uk address?

Reply:
Good point, I'll add this in!